release.yaml

```diff
@@ -47,16 +49,16 @@ rancher-monitoring:
   - 103.0.1+up45.31.1
   - 103.0.2+up45.31.1
   - 103.0.4+up45.31.1
+  - 103.0.5-rc1+up45.31.1
 rancher-monitoring-crd:
   - 103.0.1+up45.31.1
   - 103.0.2+up45.31.1
```

Comment:
Please, keep only the `rancher-monitoring-crd` version that you're currently working on and the one that should be released in the upcoming _ Rancher v2.8.3_ Release.

Reply:
Think this oddity was fixed by previous release. So should be good now?